includes/database/schemas/class-adjustments.php

```diff
@@ -186,7 +186,7 @@ final class Adjustments extends Schema {
 		array(
 			'name'       => 'date_created',
 			'type'       => 'datetime',
-			'default'    => '', // Defaults to current time in query class
+			'default'    => '0000-00-00 00:00:00',
```

Comment:
I'm not sure we can change this default as we intentionally moved away from `0000-00-00 00:00:00` due to MySQL errors  in higher versions of MySQL, at least as I read the issue: #7578; wouldn't this change reintroduce that issue?

I'm wondering if the solution mentioned here ([changing the type to `timestamp`](https://dba.stackexchange.com/questions/132951/cant-default-date-to-current-timestamp-in-mysql-5-5)) might be an option? I see in the linked docs that it can only be used for one column per table in MySQL 5.5, but maybe it's an option?

Reply:
@robincornett we have multiple columns that need that type though, so if it is limited to a single column per table, it can't be used.

Reply:
I understand that we have multiple columns that need that type--my concern is that we intentionally moved away from the `0000-00-00 00:00:00` default because it's not allowed with the strict MySQL mode in more current versions of MySQL, which is presumably something set at a hosting level, so we're introducing a regression--so I'm pondering what the alternatives might be (including selecting a primary column to have the `timestamp` type).

Would an option be defining the database version in the base `Schema` and `Table` classes from `$wpdb->db_version()`, and conditionally setting the default to the zeros or `CURRENT_TIMESTAMP` based on that? That way, users with more current versions of MySQL get the advantages of the current timestamp, we avoid the strict issue, and users on older versions have a default that works for them and is the expected format.

Reply:
Looking at how the Core WP columns are set up for `post_modified` and `post_modified_gmt`, these changes are consistent with the posts table, so I am willing to withdraw my concerns based on that. I just want to make sure that the other changes made in the PR which changed to `CURRENT_TIME` don't need to be reverted as well--and that migrated and new orders have the correct dates assigned, etc.

Reply:
> I'm wondering if the solution mentioned here ([changing the type to `timestamp`](https://dba.stackexchange.com/questions/132951/cant-default-date-to-current-timestamp-in-mysql-5-5)) might be an option?

The range for `TIMESTAMP` values is:
> `1970-01-01 00:00:01.000000` to `2038-01-19 03:14:07.999999`

I'll be 59 years old in 2038 and will either have forgotten all of this or still be working on fixing it 👴🏻 💖